vta/hardware/chisel/src/test/scala/ComputeTests.scala

```diff
@@ -0,0 +1,913 @@
+// See LICENSE.txt for license details.
+package vta
+
+import chisel3._
+import chisel3.util._
+import chisel3.testers._
+import chisel3.iotesters.{PeekPokeTester, Driver, ChiselFlatSpec}
+
+class ComputeTests(c: Compute)(implicit val p: freechips.rocketchip.config.Parameters)
+    extends PeekPokeTester(c) {
+
+  // min
+  val insn0 = "h00000001000100010000000000000000".U // uop fetch (uop_compress=true)
```

Comment:
were these constants generated somehow? It would be nice for newcomers who want to make changes to be able to specify test cases more semantically.